Taskfile.yml

```diff
@@ -21,8 +21,8 @@ tasks:
     desc: Formatting and linting
     cmds:
       - test -z "$(gofumpt -d -e . | tee /dev/stderr)"
-      - go vet ./...
```

Comment:
removed this because go vet is already included as a step in `golangci-lint run` by default.